CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
    instead of using Thread.raise on the plugins' threads. Ref: https://github.com/elastic/logstash/pull/3895
  - Dependency on logstash-core update to 2.0
 
+# v 0.1.4
```

Comment:
@purbon 
I understand.
But he situation is little bit complicated :
- in v1 compatible version, the fix is available from 0.1.4
- in v2 compatible version, the fix is available from 2.0.2

So what should we do in CHANGELOG.md ?
Add the fix in both 0.1.4 and 2.0.2 descriptions ?